paymentsheet/src/main/java/com/stripe/android/paymentsheet/PaymentSheetViewModel.kt

```diff
@@ -258,6 +242,29 @@ internal class PaymentSheetViewModel @Inject internal constructor(
             }
         }
 
+        viewModelScope.launch {
+            intentConfirmationHandler.state.collectLatest { state ->
+                when (state) {
+                    is IntentConfirmationHandler.State.Idle,
+                    // TODO(samer-stripe): Once we move all intent confirmation into the handler, we can rely on this.
+                    is IntentConfirmationHandler.State.Confirming -> Unit
```

Comment:
I'm surprised we don't use this to update the processing value or similar. Do we need this state at all?

Reply:
We will need it. Just that EPMs, Bacs, and Google Pay also maunally manage loading state so I'm avoiding using doing it here until we add everything into `IntentConfirmationHandler` (next task).

Reply:
In general, I'm a fan of doing exactly what's necessary, and leaving future stuff to my local change sets, and future PRs where they're used.

It's easy to forget whats there and why, and I'd hate for this to remain forever.

Not blocking for this PR, but in the future I think if something is unused, we should leave it out.